### ISSUE TITLE
Failed to correct inject jar to reactor on Windows 10

### ISSUE THREAD

**Issue by giflw** [takari/takari-lifecycle#58]
Hi

I'm started to use `takari-lifecycle-plugin` on Linux, but the CI is on Windows. On Linux everything works as expected, but on Windows, after generating the module package, the next module in maven reactor doesn't see the previous module binaries.

This is my configuration for this plugin. I'm using `takari-jar` packaging.

```
<plugin>
                <groupId>io.takari.maven.plugins</groupId>
                <artifactId>takari-lifecycle-plugin</artifactId>
                <version>1.13.1</version>
                <extensions>true</extensions>
                <configuration>
                    <proc>proc</proc>
                </configuration>
</plugin>
```

Thanks

**Comment by ifedorenko** [comment]
This does not provide enough information to help you troubleshoot the problem. Ideally, we'll need small standalone example project and steps to reproduce the problem.

**Comment by giflw** [comment]
Simple project added on https://github.com/giflw-test/takari-test.

Thanks

**Comment by ifedorenko** [comment]
Can you confirm the provided test project works as expected on Linux but fails on Windows?

What exact command do you use to reproduce the problem and what directory do you run it from? `mvn clean package` from takari-test/ directory?

Any particular reason you are not using takari-jar for all projects?

**Comment by giflw** [comment]
Hi

I've retested now. Works on Linux but does not in Windows.

The command was `mvn clean install` from `takari-test` directory.

About the project, it is composed by 10 modules:
- 2 libraries: jar (using takari-jar)
- 1 EAR with 3 WAR and 1 EJB
- other separated 2 WAR
- 1 JSE command line app with Java + Kotlin

As it is big, I'm trying to use takari-jar on the simplest part to see how it performs. I wish to use it in all project, but I didn't found war support or how to use to generate EJB or EAR. Also, it doesn't work with kotlin-maven-plugin.

**Comment by ifedorenko** [comment]
Looks like takari-lifecycle is broken on Windows again, it can't find any sources in `lib/src/main/java`.

We don't use Windows and nobody come forward to setup CI build on that platform, so this kind breakage sadly happens.

**Comment by giflw** [comment]
Is there a way to use AppVeyor to build on Windows, as this project is OSS?

**Comment by ifedorenko** [comment]
Yes, this is an open source project, licensed under EPL. I am not familiar with AppVeyor and won't be able to help there.

**Comment by ifedorenko** [comment]
Windows compat should be fixed in version 1.13.3.
